### ISSUE TITLE
Add support for multiple-hit signatures

### ISSUE THREAD

**Issue by radare** [radare/radare2#3448]
if a flag already exists with that name, it should append a numeric value or some kind of distinction, instead of overriding the previous signature hit. This can be useful when looking for patterns like /x does right now. So it's in fact an enhacement for the specific case of zignaturez.

**Comment by jroimartin** [comment]
Already fixed in branch https://github.com/jroimartin/radare2/tree/zign-issues. I'll do a PR fixing this and #5688, as I'm working on both issues in the same branch.

**Comment by jroimartin** [comment]
Implemented in PR #6924 .

Before:

```
$ r2 ../bins/elf/analysis/zigs_stripped
 -- r2 is meant to be read by machines.
[0x00400440]> zb test e8........
[0x00400440]> z/
Ranges are: 0x00400000 0x00601038
fs sign 7 matching function signatures
f sign.sys.b.test @ 0x004003f0
f sign.sys.b.test @ 0x00400464
f sign.sys.b.test @ 0x004004c9
f sign.sys.b.test @ 0x004004fd
f sign.sys.b.test @ 0x00400553
f sign.sys.b.test @ 0x0040056f
f sign.sys.b.test @ 0x004005ac
fs *
```

After:

```
$ r2 ../bins/elf/analysis/zigs_stripped
 -- This software comes with no brain included. Please use your own.
[0x00400440]> zb test e8........
[0x00400440]> z/
Ranges are: 0x00400000 0x00601038
fs sign 6 matching function signatures
f sign.sys.b.test_0 @ 0x004003f0
f sign.sys.b.test_1 @ 0x00400464
f sign.sys.b.test_2 @ 0x004004c9
f sign.sys.b.test_3 @ 0x004004fd
f sign.sys.b.test_4 @ 0x00400553
f sign.sys.b.test_5 @ 0x0040056f
f sign.sys.b.test_6 @ 0x004005ac
fs *
```
